commcare/src/main/java/org/motechproject/commcare/domain/CaseJson.java

```diff
@@ -42,101 +42,105 @@
     @SerializedName("case_id")
     private String caseId;
 
-    public String getDateClosed() {
-        return this.dateClosed;
-    }
+    @SerializedName("closed")
```

Comment:
Yes + a fix of getter name for caseProperties - getProperties() -> getCaseProperties()